windows/src/developer/TIKE/oskbuilder/TouchLayoutDefinitions.pas

```diff
@@ -92,15 +92,42 @@   TJSONDef = record
   TTouchLayoutPredefinedLayer = string;
 
 const
-  PredefinedLayers: array[0..7] of TTouchLayoutPredefinedLayer = (
+  PredefinedLayers: array[0..23] of TTouchLayoutPredefinedLayer = (
     'default',
+
+    // Legacy predefined layer names
     'shift',
     'ctrl',
     'alt',
     'ctrlalt',
     'ctrlshift',
     'altshift',
-    'ctrlaltshift'
+    'ctrlaltshift',
+
+    // 10.0 predefined layer names
```

Comment:
I assume these are mutually exclusive sets, and that 'shift' and 'default' will still be selectable for chiral keyboards?

Reply:
Yes; these are used only for merging desktop visual keyboards with touch keyboards (during an import). The legacy names are left for existing keyboards